tools/mvicuna.py

```diff
@@ -63,6 +63,34 @@ def rmdup(self, inPair, outPair, outUnpaired=None):
         for tmpfname, outfname in zip(tmp2OutPair, outPair):
             shutil.copyfile(tmpfname, outfname)
 
+    def rmdup_single(self, inFastq, outFastq):
+        """
+        Run mvicuna's duplicate removal operation on single-end input reads in
+            fastq format, producing output in fastq format.
+        Notes on weird behaviors of M-Vicuna DupRm:
+            For some reason, it requires you to specify both -opfq and -drm_op even for
+            single-end input.
+
+            Actually, more interestingly, this doesn't work at all.
```

Comment:
I tried valiantly... maybe a bug or something, maybe it was never tested in that scenario. I figure a nop is better than a failure.